dstep/Configuration.d

```diff
@@ -18,42 +18,71 @@ struct Configuration
     /// array of file names to translate to D
     string[] inputFiles;
 
-    /// expected programming language of input files
+    @("language|x", "Treat subsequent input files as having type <language>.")
     Language language;
 
+    @("version", "Show dstep version.")
+    bool dstepVersion;
+
+    @("clang-version", "Show libclang version.")
+    bool clangVersion;
+
     /// array of parameters needed to be forwarded to clang driver
     string[] clangParams;
 
     /// output file name or folder (in case there are many input files)
     string output;
 
-    /// strip all comments while translating
-    bool noComments;
-
-    /// package name
+    @("package", "Use <package> as package name.")
     string packageName;
 
-    /// use public imports for submodules
-    bool publicSubmodules;
+    @("comments", "Translate comments [default: true].")
```

Comment:
Hmm, I would prefer not to have a default value for those. Either revert back to the old implementation. Or add a new UDA for this, or add an additional value to the existing UDA. Not sure if it's best to be opt-in or opt-out.